contributing/commenting_fields_enums.adoc

```diff
@@ -0,0 +1,54 @@
+= Commenting on fields and enums
+
+The reference documentation of OSI is created using _Doxygen_.
+The content of the reference documentation is generated from comments in the `.proto` files.
+Follow these steps when creating comments on fields and enums to keep the documentation style consistent.
+
+**Prerequisites**
+
+* You have created a field or enum.
+
+**Steps**
+
+. Add a description to the field or enum followed by an empty comment line.
+. Add the unit of the field using the `\unit` keyword followed by an empty comment line.
+. Optionally, add literary references to the comments.
```

Comment:
I think the keyword is missing here.

Reply:
There is no keyword, but you are right, there was something missing. 
Added the missing content.